content/docs/operating/configuration.md

```diff
@@ -399,7 +399,7 @@ Prometheus relabeling to control which instances will actually be scraped. Also
 by default all apps will show up as a single job in Prometheus (the one specified
 in the configuration file), which can also be changed using relabeling.
 
-### Zookeeper Serverset SD configurations `<serverset_sd_config>`
+### `<serverset_sd_config>`
```

Comment:
The filter that generates the ToC supports `data-nav-title` attributes on the headers, which will be used in the ToC rather than the full header title. The Nanoc site uses it on e.g. the [Troubleshooting](http://nanoc.ws/doc/troubleshooting/) page ([raw source](https://raw.githubusercontent.com/nanoc/nanoc.ws/master/content/doc/troubleshooting.md)).

It doesn’t seem like this works with RedCarpet, though. The Nanoc site uses Kramdown which supports it.